packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -111,5 +111,10 @@ export const messages = {
     'A refresh of your sObject definitions is already underway. If you need to restart the process, cancel the running task.',
   force_project_create_text: 'SFDX: Create Project',
   force_project_create_open_dialog_create_label: 'Create Project',
-  force_apex_trigger_create_text: 'SFDX: Create Apex Trigger'
+  force_apex_trigger_create_text: 'SFDX: Create Apex Trigger',
+  force_start_apex_debug_logging:
+    'SFDX: Turn on Apex Debug Log for Replay Debugger',
```

Comment:
Normally "on" wouldn't be capitalized because it's a preposition that's four characters long or shorter, but we capitalize short prepositions when they're part of a term (Log In, Turn On, etc.). So, "On" in this message and "Off" in the `force_stop_apex_debug_logging` message should be capitalized.